group1/build.bat

```diff
@@ -1,7 +1,7 @@
 @echo off
 
 @if not [%1]==[] goto main
-@echo Usage: %0 ^<Version^>
+@echo Usage: %0 ^<Version^> [f]
```

Comment:
It took me a moment to realize that this `f` flag meant `force` (i.e. overwrite outputs even if they exist). Could this be documented in `README.md`?

Reply:
I didn't actually mean to push this commit to this branch 😂 But yes, I've updated README now.